bitcoin/chainparams.c

```diff
@@ -63,7 +63,7 @@ const struct chainparams networks[] = {
 				      0x5b, 0xbf, 0x28, 0xc3, 0x4f, 0x3a, 0x5e,
 				      0x33, 0x2a, 0x1f, 0xc7, 0xb2, 0xb7, 0x3c,
 				      0xf1, 0x88, 0x91, 0x0f}}}},
-     .rpc_port = 18332,
+     .rpc_port = 18443,
```

Comment:
Actually, we don't use this field (which is why this bug does nothing)  Please delete it altogether!

Reply:
Thanks for comment, I see your point, just that I use `rpc_port` now in that network-dependent-port code which I added into `lightningd.c`. And just wanted to keep the mask exactly as it is in Bitcoin Core (10111 is the difference between the mainnet and regtest).

Reply:
10111 decimal, though it may look binary :)

Reply:
https://github.com/jsarenik/lightning/blob/jsn/default_ports/lightningd/lightningd.c#L954